packages/@react-aria/utils/src/scrollIntoView.ts

```diff
@@ -63,3 +71,33 @@ function relativeOffset(ancestor: HTMLElement, child: HTMLElement, axis: 'left'|
   }
   return sum;
 }
+
+export function scrollIntoViewport(targetElement: Element, containingElement?: Element) {
+  // If scrolling is not currently prevented then we aren’t in a overlay nor is a overlay open, just use element.scrollIntoView to bring the element into view
+  if (!isScrollPrevented) {
+    let {left: originalLeft, top: originalTop} = targetElement.getBoundingClientRect();
+
+    // TODO: For some reason in the useSelectableList stories, originalTop and newTop differ by 1px, causing us to unecessarily
+    // scroll the containing element even though it may already be fully in view
+    // This is because the scrollable parent (ul) has a border of 1 that partially obscures the focused item even after scrollIntoView is called...
+    // Perhaps modify scrollIntoView calculations to take into account the borders, bleh...
```

Comment:
Does changing `box-sizing` to `border-box` have any effect?

Reply:
Mm, good idea, I tried applying `box-sizing: border-box` to the story's `ul` but unfortunately it still reproduces the issue. Lemme poke around some more

Reply:
Using `let maxY = y + scrollView.clientHeight;` in `scrollIntoView` gets us pretty close but takes into account both top/bottom borders which borks the calculations a bit